sdks/python/apache_beam/ml/gcp/visionml_test_it.py

```diff
@@ -0,0 +1,75 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# pytype: skip-file
+
+from __future__ import absolute_import
+
+import unittest
+
+from nose.plugins.attrib import attr
+
+import apache_beam as beam
+from apache_beam.ml.gcp.visionml import AnnotateImage
+from apache_beam.testing.test_pipeline import TestPipeline
+from apache_beam.testing.util import assert_that
+from apache_beam.testing.util import equal_to
+
+try:
+  from google.cloud.vision import types
+except ImportError:
+  types = None
+
+
+def extract(response):
+  for r in response.responses:
+    for text_annotation in r.text_annotations:
+      yield text_annotation.description
+
+
+@attr('IT')
+class VisionMlTestIT(unittest.TestCase):
```

Comment:
LGTM. 

One small question though: Should we guard against environments where vision isn't installed by having the `unittest.skipIf...`?

Reply:
Fair enough.
The test will always be skipped if `TestPipeline(is_integration_test=True)` is present and `--test-pipeline-options` is not provided (it's only provided in gcp environments). But maybe more explicit guard could be useful.

Reply:
I see. Yeah fair either way then. LGTM :)